src/BowlingGame.js

```diff
@@ -0,0 +1,66 @@
+var BowlingGame = function () {
+  this.frames = [];
+  this.totalScore = 0;
+  this._gameSetup();
+};
+
+BowlingGame.prototype.totalScoreBonuses = function () {
+  this._frameScoreBonuses();
+  var allBonuses = this.frames.reduce(function (totalBonus, frameBonus) {
+    return totalBonus + frameBonus.bonusScore;
+  }, 0);
+  return allBonuses;
+};
+
+BowlingGame.prototype.totalBaseScore = function () {
+  this._framesBaseScores();
+  var allScores = this.frames.reduce(function (totalScore, frameScore) {
```

Comment:
Could the code have a `sum` function that could take an array of numbers and return a sum? This function could be used by `totalBaseScore` and `totalScoreBonuses`.